src/test/Settings/FeatureLevels.test.tsx

```diff
@@ -75,15 +75,19 @@ describe('<FeatureLevels/>', () => {
     expect(ul).toBeInTheDocument();
     expect(ul).toBeVisible();
 
-    await user.selectOptions(ul, testT(FeatureLevel[FeatureLevel.BETA], 'common'));
+    await user.selectOptions(ul, testT(FeatureLevel[FeatureLevel.BETA], { ns: 'common' }));
 
-    expect(ul).not.toBeInTheDocument(); // Should close menu
+    await expect(ul).not.toBeInTheDocument(); // Should close menu
 
-    const betaOption = screen.getByText(testT(FeatureLevel[FeatureLevel.BETA], 'common'));
+    const betaOption = await screen.findByText(
```

Comment:
Was there an issue with this? Ideally, this line:

```ts
 await user.selectOptions(ul, testT(FeatureLevel[FeatureLevel.BETA], { ns: 'common' }));
```

should flush out all tasks and correctly update component state